.github/workflows/appimage.yml

```diff
@@ -1,8 +1,20 @@
-name: Build AppImage SLAM Server
+name: Build AppImage Cartographer
 
 on:
   workflow_dispatch:
+    inputs:
+      release_type:
+        required: true
+        description: Release version for this appimage (latest/rc/stable)
```

Comment:
@EshaMaharishi let me know if you agree with this description, and more importantly the default

Reply:
This looks good!